### PR TITLE
Rename RunOrStop as ControlMessage

### PR DESCRIPTION
**Pull request by coot**
This patch is pulled out from `coot/connection-manager` branch to avoid future
merge conflicts.

### CODE REVIEW

File: ouroboros-network/src/Ouroboros/Network/KeepAlive.hs [hunk 1]
```diff
@@ -47,27 +47,29 @@ keepAliveClient
...
     decisionSTM delayVar = do
-       shouldStop <- shouldStopSTM
-       case shouldStop of
-            Stop -> return Stop
-            Run  -> do
+       controlMessage <- controlMessageSTM
+       case controlMessage of
+            Terminate -> return Terminate
+
+            -- Contitnue
```
Comment (mrBliss) [inline ouroboros-network/src/Ouroboros/Network/KeepAlive.hs:68]:
```suggestion
            -- Continue
```

File: ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+        fixRanges :: Int -> Int -> Int -> Maybe (Int, Int)
+        fixRanges l n m | n + m < l      = Just (n, m)
+                        | n > 1 && m > 1 = fixRanges (pred n) (pred m) l
+                        | n > 1          = fixRanges (pred n) m l
+                        | m > 1          = fixRanges n (pred m) l
+                        | otherwise      = Nothing
+
+
+
+-- | Run two 'tipSampleClient''s against two servers  using 'Channels', returns
```
Comment (mrBliss) [inline ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs:145]:
```suggestion
-- | Run two 'tipSampleClient's against two servers  using 'Channels', returns
```

File: ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DerivingVia         #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections       #-}
+
+module Test.Ouroboros.Network.TipSample.Client (tests) where
```
Comment (mrBliss) [inline ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs:11]: Is this file intended to be part of this PR? I don't see it being added to the cabal file nor added to the lists of tests to run.

Comment (coot) [reply ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs:11]: thanks for catching this, no it should not.

**Review by mrBliss — APPROVED** [review]

**Comment by coot** [comment]
bors merge

**Comment by coot** [comment]
bors cancel

**Comment by mrBliss** [comment]
bors merge
